tests/Database/DatabaseEloquentBelongsToManyTest.php

```diff
@@ -116,7 +116,7 @@ public function testModelsAreProperlyMatchedToParents()
         $this->assertEquals(2, $models[1]->foo[0]->pivot->user_id);
         $this->assertEquals(2, $models[1]->foo[1]->pivot->user_id);
         $this->assertEquals(2, count($models[1]->foo));
-        $this->assertEquals(0, count($models[2]->foo));
+        $this->assertEquals(0, count((array) $models[2]->foo));
```

Comment:
> Actually only array and objects that implement countable can be used with count

In PHP 7.2, those are the types considered countable, yes, `array` and `Countable`. In 7.2, `null` was also countable.

> I'm 👍 for changing it on 5.5

Seems like a fair change perhaps. Ping @taylorotwell.